framework/src/systems/NonlinearSystemBase.C

```diff
@@ -589,7 +592,6 @@ NonlinearSystemBase::computeResidualTags(const std::set<TagID> & tags)
     zeroTaggedVectors(tags);
     computeResidualInternal(tags);
     closeTaggedVectors(tags);
-
```

Comment:
Is clang-format removing these blank lines? If not, I really don't think they should be part of this diff...

Reply:
I think I accidentally removed this line. I have added this line back.